### PR TITLE
Add default_columns classmethod to Progress class

### PR DESCRIPTION
**Pull request by ptmcg**
## Type of changes

- [ ] Bug fix
- [x] New feature
- [ ] Documentation / docstrings
- [ ] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/psf/black) with default args on new code.
- [ ] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [x] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

Added new default_columns() classmethod to the Progress class, so that client code does not need to replicate the defaults literally, but can just add `*Progress.default_columns()`

### CODE REVIEW

File: rich/progress.py [hunk 1]
```diff
@@ -613,6 +608,15 @@ def __init__(
         self.print = self.console.print
         self.log = self.console.log
 
+    @classmethod
+    def default_columns(cls) -> Tuple[ProgressColumn, ...]:
+        return (
```
Comment (willmcgugan) [inline rich/progress.py:613]: Could you change this to `get_default_changes`  and add a brief docstring...

**Review by willmcgugan — CHANGES_REQUESTED** [review]
Great! Just a request for name and docstring.

File: rich/progress.py (hunk 1 above)
Comment (ptmcg) [reply rich/progress.py:613]: Did you mean get_default_columns?

Comment (willmcgugan) [reply rich/progress.py:613]: Yeah, `get_default_columns`.  Another case of the Mondays...

Comment (ptmcg) [reply rich/progress.py:613]: I've got a decent docstring, but just want to double-check on the name. Do you *really* want get_default_columns()? I usually associate getters with instances, not the class. For this method, I'm not sure what value the "get_" prefix adds.

Comment (willmcgugan) [reply rich/progress.py:613]: Leaning towards `get`. Implies it does something more than expose data.

Comment (ptmcg) [reply rich/progress.py:635]: See updates

**Comment by ptmcg** [comment]
This should be ready whenever you are. Unless you have further comments, I'll assume this is in your queue for the next release.

**Comment by willmcgugan** [comment]
Thanks!

**Pull request merged**